### ISSUE TITLE
Add option to display only standard/only recurrent entries with `list`

### ISSUE THREAD

**Issue by pylipp** [pylipp/financeager#51]
e.g. by extending the functionality of the `--filter` option, or by providing a new one.

**Comment by pylipp** [comment]
Rather: list info recurrent entries in tabular form by running `list --recurrent`

```
| ID | Name | Value | Frequency | Start      | End |
|  1 | Rent |  -500 | monthly   | 2020-01-01 | --- |
```

**Comment by pylipp** [comment]
Closed in v0.26.3.0
